src/components/GameView.tsx

```diff
@@ -35,13 +36,15 @@ const useStyles = makeStyles(theme => ({
 
 type Props = {
   chosenSongMIDI: any;
+  myPart?: Part | null;
```

Comment:
I think the question mark should be sufficient
```suggestion
  myPart?: Part;
```

Reply:
ohh or will it be null? just to check, the players must have selected part before entering the game view and the tracks will be present as well right